services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -202,6 +205,18 @@ export default class IndexPage extends React.Component<Props> {
   }
 
   renderCommission(commission: Commission) {
-    return <li key={commission.id}>{commission.name}</li>;
+    return (
+      <li style={{ listStyleType: 'none' }}>
+        {' '}
+        <Checkbox
+          id={null}
+          name="commissions"
+          value="commission"
+          onChange={null}
+          onBlur={null}
+        />{' '}
+        <span className="cb-l">{commission.name}</span>
```

Comment:
This should be rendered in the `Checkbox` component, just as the label is rendered in the `TextInput` component.